fcrepo-transform/src/main/java/org/fcrepo/transform/TransformationFactory.java

```diff
@@ -56,7 +59,7 @@ public TransformationFactory() {
     public <T> Transformation<T> getTransform(final MediaType contentType, final InputStream inputStream) {
         final String mimeType = contentType.toString();
         if (mimeToTransform.containsKey(mimeType)) {
-            return (Transformation<T>) mimeToTransform.get(contentType.toString()).newTransform(inputStream);
+            return mimeToTransform.get(contentType.toString()).apply(inputStream);
```

Comment:
What was perplexing me is the fact that `Transformation` has an `apply(RdfStream)` method that overrides `Function::apply`. However, there is another lambda function at play that has its own `apply` method.
I see the layers now.